docs/install.md

```diff
@@ -71,7 +71,7 @@ helm install ingress-apisix -n ingress-apisix ./charts/ingress-apisix
 ## Master branch builds
 
 ```shell
-docker build -t apache/ingress-controller:v0.1.0 ../.
+cd .. && make build-image
```

Comment:
why do we need `cd .. &&` here?

Reply:
@gxthrj

Reply:
The file `install.md` is in the `docs` path, but `Makefile` is in the root directory.

Reply:
We don't need to edit commands from the perspective of the path of `install.md`.

```shell
$ cd /path/to/apisix-ingress-controller
$ make build-image
```

I think that's enough.

Reply:
OK